src/components/tree-item/tree-item.tsx

```diff
@@ -230,6 +241,22 @@ export class TreeItem implements ConditionalSlotComponent, InteractiveComponent
     ) : null;
 
     const hidden = !(this.parentExpanded || this.depth === 1);
+    const { hasEndActions } = this;
+    const slotNode = (
+      <slot
+        key="actionsEndSlot"
+        name={SLOTS.actionsEnd}
+        onSlotchange={this.actionsEndSlotChangeHandler}
+      />
+    );
+    const iconStartEl = (
+      <calcite-icon
+        class="calcite-tab-title--icon icon-start"
```

Comment:
can you add these to the CSS const in the resources file and use them from that?

```jsx
class={{[CSS.tabTitle]: true, [CSS.iconStart]: true}}
```

Reply:
Perfect!